deployment/openshift-play-scala-grpc-example.yml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: "apps/v1beta2"
+kind: Deployment
+metadata:
+  name: "play-scala-grpc-example-v1-0-snapshot"
+  labels:
+    appName: "play-scala-grpc-example"
+    appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+  template:
+    metadata:
+      labels:
+        appName: "play-scala-grpc-example"
+        appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: "play-scala-grpc-example"
+          image: "docker-registry-default.centralpark.lightbend.com/play-scala-grpc-example/play-scala-grpc-example:1.0-SNAPSHOT"
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: "JAVA_OPTS"
+              value: "-Dplay.http.secret.key=a-very-strong-key-for-production -Dplay.filters.hosts.allowed.0=myservice.example.org -Dplay.server.pidfile.path=/dev/null"
```

Comment:
pidfile disabled because https://github.com/lightbend/tooling-team/issues/238

Reply:
Disabling the use of the pidFile is only needed on OpenShift ATM (until file [permissions are configurable](https://github.com/sbt/sbt-native-packager/pull/1193)).

Still, using pidFile in an ephemeral container doesn't make much sense so I thin we should suggest disabling it in all deployment targets (vanilla k8s, openshift, ...).

WDYT @jroper ?

Reply:
Yes it makes sense to disable. Also, I'd prefer that we create a separate configuration file for prod, and put all the prod specific configuration in there, including PID file. Then the only command line argument that needs to be passed here is the configuration file name.

Reply:
Ah, I like that approach, It'll make the docs a lot cleaner.

I'll consolidate the settings into `prod-application.conf` on a separate PR once both this and #38 are merged.

Reply:
raised https://github.com/playframework/play-scala-grpc-example/issues/41